tests/zfs-tests/tests/functional/acl/posix/posix_001_pos.ksh

```diff
@@ -54,8 +62,12 @@ if [ "$?" -eq "0" ]; then
 	# Should be able to write to file
 	log_must user_run $ZFS_ACL_STAFF1 "echo 'echo test > /dev/null' > $TESTDIR/file.0"
```

Comment:
We should really be checking the permission bits for these tests too to verify this is succeeding for the reasons we thing.